tools/python/boutiques/tests/test_desc2func.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+import sys
+from boutiques.descriptor2func import function
+if sys.version_info < (2, 7):
+    from unittest2 import TestCase
+else:
+    from unittest import TestCase
+
+
+class TestDesc2func(TestCase):
+
+    def test_desc2func(self):
```

Comment:
can we also test the exec mode? Just a sanity check that things are being passed properly

Reply:
done